sys/aarch64/trap.c

```diff
@@ -167,3 +174,23 @@ void kern_trap_handler(ctx_t *ctx) {
       kernel_oops(ctx);
   }
 }
+
+__no_profile void kern_exc_handler(ctx_t *ctx, kexc_reason_t reason) {
```

Comment:
Is it possible to get the reason from `exc_code` as it is done in MIPS?

Reply:
I believe it isn't. In Aarch64 traps are vectored and I would expect one to distinguish the cause based on the vector being used.